backend/functions/user.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+function login($data,$con) {
+
+    $users = selectStatement($con,'user',false,null,null);
+
+    foreach ($users as $userData) {
```

Comment:
er hoort maar 1 of 0 users te returnen dus er zou in theorie geen foreach nodig moeten zijn.

Reply:
aangezien email uniek moet zijn (dus dan kan je op die WHERE gebruiken)